src/index.js

```diff
@@ -5,7 +5,13 @@ import {
   XRLayer
 } from './layers';
 import { VivViewer, PictureInPictureViewer, SideBySideViewer } from './viewers';
-import { VivView, OverviewView, DetailView, SideBySideView, getDefaultInitialViewState } from './views';
+import {
+  VivView,
+  OverviewView,
+  DetailView,
+  SideBySideView,
+  getDefaultInitialViewState
```

Comment:
Is there a different prettier config? Curious why this line jumped down. NBD, just curious.

Reply:
Hmm, I don't think anything changed.  Maybe because all of the other lines are now going down instead of across this one changed too?

Reply:
I'll keep an eye on it and if something like this happens again, I'll flag it and figure out what happened.